src/help/guides/adding-repos.md

```diff
@@ -5,24 +5,36 @@ lang: en-US
-1. Tap "**Add**"
-1. Select the repository that you have added, which should have appeared in the list of repositories on this page
-    > If you do not see sources, you may have typed in your Base URL incorrectly. Delete the repository and try again. If you do see sources, congratulations!
+1. Visit [Known Repositories](/help/guides/adding-repos/#known-repositories) on your Device's web browser.
+2. Add a Source Repository by tapping "**Add to Paperback**".
+
+## Installing a Source from a repository
+1. Follow [Adding an External Repository](/help/guides/adding-repos/#adding-an-external-repository) to add a source repository to Paperback.
+2. Navigate to the **Settings** tab then tap **Sources** menu item.
+3. Tap on the previously added repository under the "Repositories" section.
+4. Choose a source to install.
 
 ## Known Repositories
 This is a list of all of the known repositories currently for Paperback, as well as the repository contents.
-| Repository Name | Base URL | Available Sources | Notes | Add Repo (iOS Only) |
-| :-------------: | :------: | :---------------- | :---: | :---:
-| **Primary Sources**   | https://paperback-ios.github.io/extensions-beta | <ExtensionsList url="https://paperback-ios.github.io/extensions-beta"/> | Officialy maintained repository, contains a few major sources suited for most users | [Add to Paperback](paperback://addRepo?displayName=Primary%20Sources&url=https://paperback-ios.github.io/extensions-beta) |
-| **H-Extensions**      | https://paperback-ios.github.io/h-extensions | <ExtensionsList url="https://paperback-ios.github.io/h-extensions"/> | Official repository for 18+ sources. These also include redirector versions of the source, which allow you to view content even if it is blocked in your country, without a VPN | [Add to Paperback](paperback://addRepo?displayName=H-Extensions&url=https://paperback-ios.github.io/h-extensions) |
-| **MangaDex Unlocked** | https://pogogo007.github.io/extensions-beta | <ExtensionsList url="https://pogogo007.github.io/extensions-beta"/> | This source overwrites MangaDex and unlocks the full contents of the website. No content is filtered. May load faster than the built-in version of the source | [Add to Paperback](paperback://addRepo?displayName=MangaDex%20Unlocked&url=https://pogogo007.github.io/extensions-beta) |
-| **Beta Extensions**   | https://conradweiser.github.io/extensions | <ExtensionsList url="https://conradweiser.github.io/extensions"/> | These extensions are upcoming to the official repository. These sources may or may not work | [Add to Paperback](paperback://addRepo?displayName=Beta%20Extensions&url=https://conradweiser.github.io/extensions) |
+
+<div>
+    <ExtensionsList
+        url="https://paperback-ios.github.io/extensions-beta"
+        name="Primary Sources"
+        description="Officialy maintained repository, contains a few major sources suited for most users"/>
```

Comment:
It could be great to either add a space or remove the one from the other repositories
```suggestion
        description="Officialy maintained repository, contains a few major sources suited for most users" />
```